zip-0317.rst

```diff
@@ -451,27 +528,13 @@ threshold. This is no longer expressible with the formula specified above.)
 Endorsements
 ============
 
-The following entities/groups/individuals expressed their support for the
-updated fee mechanism:
-
-*Developer Groups or Sole OSS contributors*
-
-..
-  * Zecwallet Suite (Zecwallet Lite for Desktop/iOS/Android & Zecwallet FullNode)
-  * Nighthawk Wallet for Android & iOS
-
-To express and request your support to be added to this ZIP please comment
-below indicating:
-
-* (group) name/pseudonym
-* affiliation
-* contact
-
-or, conversely e-mail the same details to the Owner of the ZIP.
+The following entities and developers of the listed software expressed their
+support for the updated fee mechanism:
 
-TODO: Endorsements may depend on specific parameter choices. The ZIP
-Editors should ensure that the endorsements are accurate before marking
-this ZIP as Active.
+* Zecwallet Suite (Zecwallet Lite for Desktop/iOS/Android & Zecwallet FullNode)
+* Nighthawk Wallet for Android & iOS
```

Comment:
Instead of the Endorsements section, add Deployment subsections for Zecwallet Lite, Zecwallet FullNode, Nighthawk, etc.)

Reply:
It seems like there's nothing to report there so far, so I will merge this as-is and revisit those Deployment sections later.